ClaimOwnership/handler.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  IResponseErrorInternal,
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseSuccessJson
+} from "@pagopa/ts-commons/lib/responses";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "@pagopa/ts-commons/lib/request_middleware";
+import * as express from "express";
+import { flow, pipe } from "fp-ts/lib/function";
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import { ContextMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { RequiredParamMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/required_param";
+import {
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "@pagopa/ts-commons/lib/strings";
+import { Pool, QueryResult, QueryResultRow } from "pg";
+import { Context } from "@azure/functions";
+import { knex } from "knex";
+import { IConfig, IDecodableConfigPostgreSQL } from "../utils/config";
+import {
+  IDbError,
+  toPostgreSQLError,
+  toPostgreSQLErrorMessage
+} from "../models/DomainErrors";
+import { SubscriptionStatus } from "../GetOwnershipClaimStatus/handler";
+import { queryDataTable } from "../utils/db";
+import { OrganizationQueueItem } from "./types";
+
+type Handler = (
+  context: Context,
+  organizationFiscalCode: OrganizationFiscalCode,
+  delegateId: NonEmptyString
+) => Promise<IResponseSuccessJson<void> | IResponseErrorInternal>;
+
+export const updateSqlSubscription = (dbConfig: IDecodableConfigPostgreSQL) => (
```

Comment:
I think the function name is misleading: it let me think it updates subscription data; instead, I see it _sets migration status to PENDING for **every** subscription of a Delegate for an Organization_.

(also, a comment might be helpful for the reader to understand what to expect without digging into the function implementation details)

Reply:
You're right, it's done by f129126